lib/pure/math.nim

```diff
@@ -865,6 +865,18 @@ func floorDiv*[T: SomeInteger](x, y: T): T =
   let r = x mod y
   if (r > 0 and y < 0) or (r < 0 and y > 0): result.dec 1
 
+func `div`*[T: float32|float64](x, y: T): T =
+  ## Computes the division operation for float values.
```

Comment:
Which "division"? ``floor(x / y)``?

Reply:
trunc(default) division just like `mod(x, y: float)`.

>   func `mod`*(x, y: float32): float32 {.importc: "fmodf", header: "<math.h>".}
>  func `mod`*(x, y: float64): float64 {.importc: "fmod", header: "<math.h>".}
    ## Computes the modulo operation for float values (the remainder of ``x`` divided by ``y``).

Reply:
Ah, I put it at the wrong place.

Reply:
The implementation for Python floor(float) div
https://github.com/python/cpython/blob/52a327c1cbb86c7f2f5c460645889b23615261bf/Objects/floatobject.c#L636

I verified floor div for integer between Python and Nim, they are the same.
such as `floorDiv(5.6, 2.5)`
And `floorDiv` is based on `div`.